src/backend/distributed/transaction/backend_data.c

```diff
@@ -502,7 +502,7 @@ UserHasPermissionToViewStatsOf(Oid currentUserId, Oid backendOwnedId)
 		return true;
 	}
 
-	if (is_member_of_role(currentUserId,
+	if (has_privs_of_role(currentUserId,
```

Comment:
Was this accidentally pulled in? Why change from is_member_of_role to has_privs_of_role?